stanford/bin/certs-generate-course

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+_required_variables='REMOTE_BOX COURSE_ID SYSTEM_USER'
```

Comment:
These should be set by jenkins, but you can set them manually when running directly via command line.